lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -312,10 +315,25 @@ def get(self, request, *args, **kwargs):
                     )
                 ] if 'children' in chapter_data else []
 
+        # Get has_scheduled_content data
+        collected_block_structure = get_block_structure_manager(course_key).get_collected()
```

Comment:
You don't need to grab this and pass it in. If you don't pass it in, lower level code will get it themselves. This was only separately called in the progress view because the block structure was also passed to some other code. So it was an optimization there.